app/models/participation.rb

```diff
@@ -2,5 +2,16 @@ class Participation < ApplicationRecord
   belongs_to :user
   belongs_to :playground
 
-  validates :status, inclusion: { in: %w[coming ongoing finished] }
```

Comment:
you should migrate the model motherfucker

Reply:
I'm working on it ! Wait for my next pull request.